plugins/core/tracing.go

```diff
@@ -64,6 +68,16 @@ func (t *Tracer) CreateEntrySpan(operationName string, extractor interface{}, op
 	}
 
 	span, _, err := t.createSpan0(ctx, tracingSpan, opts, withRef(ref), withSpanType(SpanTypeEntry), withOperationName(operationName))
+	if err == nil {
+		id := span.GetSegmentID()
+		t.ProfileManager.ToProfile(operationName, id)
```

Comment:
Does this mean all the endpoints will be profiling, without any endpoint name checking?